### ISSUE TITLE
bedlam training dataset

### ISSUE THREAD

**Issue by bowieshi** [fanegg/Human3R#16]
Hi! Thanks for your amazing work! May I ask something about the training dataset? I found that the `fetch_bedlam.sh` only donwload the 6fps version of bedlam processed_labels. Is that Human3R trained with 6fps bedlam dataset only? Will the HMR results be better if trained with 30 fps dataset?

**Comment by Dali936** [comment]
> Hi [@bowieshi](https://github.com/bowieshi), thanks for your interest! We train Human3R using the released BEDLAM [SMPL-X ground-truth labels](https://bedlam.is.tue.mpg.de/download.php), provided at 6 fps, along with their paired frames. If you can obtain 30 fps SMPL-X annotations paired with images or train on the latest [BEDLAM2 ](https://bedlam2.is.tuebingen.mpg.de/) dataset, I believe that would lead to better performance.

Hi, i have noticed that bedlam have the pair annotation and images in huggingface at 30fps, while it is too large. i want to know how can i get the depth maps and segmentations mask and others at 6fps.

<img width="1508" height="608" alt="Image" src="https://github.com/user-attachments/assets/af67a3ef-f86c-4390-b25c-615abfa949af" />
